itl/plugins-contrib.d/vmware.conf

```diff
@@ -12,6 +12,10 @@ template CheckCommand "vmware-esx-command" {
 			set_if = "$vmware_ignoreunknown$"
 			description = "Sometimes 3 (unknown) is returned from a component. But the check itself is ok. With this option the plugin will return OK (0) instead of UNKNOWN (3)."
 		}
+		"--maintenance_mode_state" = {
+			value = "$vmware_maintenance_mode_state$"
+			description = "Set status in case ESX host is in maintenace mode. Possible Values are: ok or OK, CRITICAL or critical or CRIT or crit, WARNING or warning or WARN or warn. Default is UNKNOWN because you do not know the real state. Values are case insensitive."
+		}
```

Comment:
This argument feels misplaced between `--ignore_unknwon` and `--ignore_warning`. Could you please move it down below `--authfile`.